src/pages/index.js

```diff
@@ -45,40 +89,106 @@ addCardMOB.addEventListener("click", () => {
   addPlacePopup.open();
 });
 
+profilePic.addEventListener("click", () => {
+  profilePicPopup.open();
+});
+
 //***************************************************************************************************************************//
 //                                             Event Handlers                                                                //
 //***************************************************************************************************************************//
 
 function handleEditProfile(userData) {
+  const submitButton = editProfileModal.querySelector(
+    "#edit-profile-modal-submit-button"
+  );
+  submitButton.textContent = "Saving...";
+  api.patchProfileInfo(userData.name, userData.job);
   userInfo.setUserInfo(userData);
   editProfilePopup.popupForm.reset();
   editProfileFormValidator.toggleButtonState();
   editProfilePopup.close();
+  submitButton.textContent = "Save";
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.